### DISCUSSION TITLE
joltc api coverage

### DISCUSSION THREAD

**Discussion by oh-facts** [amerkoleci/JoltPhysicsSharp#41] [Q&A] [answered]
Hi,
How up to date is joltc with jolt? Is there a list of features present / missing? Does joltc match a version of jolt currently? Thank you!

**Comment by amerkoleci** [comment, marked as answer]
Hi, 
Joltc is updated with latest 4.0.2 stable release of jolt physics, thus lot of bindings are missing into joltc, I add stuff when needed.

Feel free to contribute.

Cheers!

**Reply by oh-facts** [reply]
Thats amazing! I would love to contribute once I'm familiar with the API! I love what you've started here.
Cheers!
